src/hooks/use-key-press.hook.ts

```diff
@@ -20,5 +24,5 @@ export const useKeyPress = (
     return () => {
       document.removeEventListener(`keydown`, onKeyDown)
     }
-  }, [keyName, onKeyPress])
+  }, [onKeyPress, ...keyNames])
```

Comment:
Yeah, you must fix the errors here:
![image](https://github.com/ajktown/wordnote/assets/53258958/583f9132-d109-4291-873b-bf91a990095d)

Reply:
Handled in [bd174bd](https://github.com/ajktown/wordnote/pull/210/commits/bd174bd481b2b2a340a90dfd9680111fe24080eb)

Reply:
Handled in [a1f5eee](https://github.com/ajktown/wordnote/pull/210/commits/a1f5eee512f21a4c5e4dc680b645ecf5931157c3)